test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -314,18 +314,15 @@ public void close() {
    */
   private static File createTempStore(String storePath) {
     File f = null;
-    try (InputStream trustStoreIs = CcmBridge.class.getResourceAsStream(storePath)) {
-      f = File.createTempFile("server", ".store");
-      logger.debug("Created store file {} for {}.", f, storePath);
-      try (OutputStream trustStoreOs = new FileOutputStream(f)) {
-        byte[] buffer = new byte[1024];
-        int len;
-        while ((len = trustStoreIs.read(buffer)) != -1) {
-          trustStoreOs.write(buffer, 0, len);
-        }
-      }
+    try (OutputStream os = new FileOutputStream(f = File.createTempFile("server", ".store"))) {
+      Resources.copy(CcmBridge.class.getResource(storePath), os);
     } catch (IOException e) {
       logger.warn("Failure to write keystore, SSL-enabled servers may fail to start.", e);
+    } finally {
```

Comment:
Imo this could be simplified to just `f.deleteOnExit();` placed right after the try block start, since `f` cannot be null at that point. But this way is OK too.

Reply:
Actually, now that I look at it again, the file is either created or not during `createTempFile`. If that throws an Exception, the file isn't created and doesn't need `deleteOnExit()` called. If it is created, a FileOutputStream is created on top. That either throws a FileNotFoundException or a SecurityException if it fails. It would be very difficult to throw either of those if creating the temp file didn't already fail so...
TL;DR; going to do it the way you suggest. :smiley: